orbit/cmd/orbit/orbit.go

```diff
@@ -2488,3 +2450,45 @@ func executeFleetDesktopWithPermanentError(desktopPath string, errorMessage stri
 
 	return nil
 }
+
+func openBrowserWindow(browserURL string) error {
+	switch runtime.GOOS {
+	case "linux":
+		loggedInUser, err := user.UserLoggedInViaGui()
+		if err != nil {
+			return fmt.Errorf("get logged in user: %w", err)
+		}
+
+		if loggedInUser == nil {
+			return errors.New("no user logged in")
+		}
+
+		browserBin := "/usr/bin/xdg-open"
+		firefoxBin := "/usr/bin/firefox"
+		if _, err := os.Stat(firefoxBin); err == nil {
+			browserBin = firefoxBin
+		}
+
+		var opts []execuser.Option
+		opts = append(opts, execuser.WithUser(*loggedInUser))
+		opts = append(opts, execuser.WithArg(browserURL, ""))
+		log.Debug().Str("browser", browserBin).Str("url", browserURL).Str("user", *loggedInUser).Msg("opening browser for setup experience")
+		if _, err := execuser.Run(browserBin, opts...); err != nil {
+			return fmt.Errorf("opening browser with %s: %w", browserBin, err)
+		}
+	case "windows":
+		unescapedAmpsRegex := regexp.MustCompile(`([^\\^])&`)
+		browserURL = unescapedAmpsRegex.ReplaceAllString(browserURL, "${1}^&")
```

Comment:
This is the only change to the existing code, to ensure that querystrings work when opening the browser window on Windows.  The same change was made to the `open` package in the original PR.